dubbo-common/src/main/java/com/alibaba/dubbo/common/Constants.java

```diff
@@ -589,7 +589,8 @@
 
     public static final long DEFAULT_TPS_LIMIT_INTERVAL = 60 * 1000;
 
-    public static final String DECODE_IN_IO_THREAD_KEY = "decode.in.io";
+    //public static final String DECODE_IN_IO_THREAD_KEY = "decode.in.io";
+    public static final String DECODE_IN_IO_THREAD_KEY = "decodeinio";
```

Comment:
This change will cause the incompatible problem for users already used this property.

Use @Parameter annotation to solve this problem, see the comment below.